zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -47,3 +47,123 @@ pub const POST_BLOSSOM_HALVING_INTERVAL: Height =
 ///
 /// Usage: founders_reward = block_subsidy / FOUNDERS_FRACTION_DIVISOR
 pub const FOUNDERS_FRACTION_DIVISOR: u64 = 5;
+
+/// Function `FounderAddressChangeInterval` as specified in [protocol specification §7.8][7.8]
+///
+/// Rust trucates the division down, to get ceiling effect we sum 1 to the end of the calculation.
+/// We use the main network lenght as both networks are the same in size.
+///
+/// [7.8]: https://zips.z.cash/protocol/canopy.pdf#foundersreward
+
+pub const FOUNDER_ADDRESS_CHANGE_INTERVAL: u64 =
+    ((SLOW_START_SHIFT.0 + PRE_BLOSSOM_HALVING_INTERVAL.0)
+        / FOUNDERS_REWARD_ADDRESSES_MAINNET.len() as u32) as u64
+        + 1;
```

Comment:
`truncate(n/d) + 1` is 1 more than `ceiling(n/d)` when `n/d` is an integer.

Let's use `num_integer::div_ceil` instead?
(`div_ceil` is not a constant function, so `FOUNDER_ADDRESS_CHANGE_INTERVAL` will need to be turned into a function.)

Here are some changes I have made in the suggestion:
- [x] use the Height type and u32, because an interval is a height
- [x] use a constant for the fixed address list size
- [x] use the standard integer ceiling division function: `ceil_div(n, d) = (n + d - 1) / d`

Here are some changes I'd like you to make:
- [ ] add the `num_integer` crate as a dependency
- [ ] move this function to `subsidy::founders_reward`
- [ ] add a test that makes sure that the lists for both networks are `FOUNDERS_ADDRESS_COUNT` long
- [ ] add a test that fails with `truncate(n) + 1` but succeeds with `ceiling(n)`

```suggestion
/// The number of founders reward address intervals, for both mainnet and testnet.
pub const FOUNDERS_ADDRESS_COUNT: u32 = 48;

/// Function `FounderAddressChangeInterval` as specified in [protocol specification §7.8][7.8]
///
/// [7.8]: https://zips.z.cash/protocol/canopy.pdf#foundersreward
pub fn founders_address_change_interval() -> Height {
    use num_integer::div_ceil;
    let interval = div_ceil(SLOW_START_SHIFT.0 + PRE_BLOSSOM_HALVING_INTERVAL.0, FOUNDERS_ADDRESS_COUNT);
    Height(interval)
}
```

Reply:
Thanks @teor2345 i applied everything at https://github.com/ZcashFoundation/zebra/pull/1170/commits/bfccd335f654546c18d57fdd954b348a2446bf2f